modules/operations/review.js

```diff
@@ -9,7 +9,8 @@ import {
 
 export function operationReview(selectedIDs, context) {
     const entityId = selectedIDs[0];
-    const entityLoc = getLocation(context.hasEntity(entityId));
+    let members = context.hasEntity(entityId).members;
+    const entityLoc = members ?  getLocation(context.hasEntity(members[0].id)) : getLocation(context.hasEntity(entityId));
```

Comment:
I found another edge case where this code will have to filter out relations or traverse down relations if the first member of the relation is also a relation.  You can reproduce with the attached file.  Select the point feature and then select it's parent "site" relation.
[site_relation.zip](https://github.com/ngageoint/hootenanny-ui/files/11501155/site_relation.zip)